### PR TITLE
Updating PayableInvoices documentation

### PR DESCRIPTION
**Pull request by kamundson-va**
Updating documentation to reflect changes based on feedback on https://github.com/vendasta/api-gateway/pull/10

@vendasta/lucky-charms @richard-rance @vendasta/psycho-billers

### CODE REVIEW

File: openapi/platform/platform.yaml [hunk 1]
```diff
@@ -708,12 +708,12 @@ paths:
       tags:
         - options
         - partners
-  /wholesaleInvoices:
+  /payableInvoices:
```
Comment (coreyhickson) [inline openapi/platform/platform.yaml:711]: Should we take this a step further and go with `accountsPayableInvoices`?

**Review by coreyhickson — APPROVED** [review]
Just wondering if we're saying Payable if we want to say the whole AccountsPayable or not

File: openapi/platform/platform.yaml (hunk 1 above)
Comment (kamundson-va) [reply openapi/platform/platform.yaml:711]: That makes sense to me. I guess there are just some open questions that remain around the naming on the RFC, so I'll try to resolve those.

File: openapi/platform/platform.yaml [hunk 2]
```diff
@@ -1135,19 +1134,19 @@ components:
...
         type:
           type: string
           enum:
-            - wholesaleInvoices
+            - payableInvoices
         attributes:
           type: object
           properties:
-            createdAt:
+            created:
```
Comment (richard-rance) [inline openapi/platform/platform.yaml:1149]: Please keep this as `createdAt`. We want to establish a convention of strings that are formatted as `date-time` ending with `At`.

File: openapi/platform/platform.yaml (hunk 1 above)
Comment (richard-rance) [reply openapi/platform/platform.yaml:711]: Is there an advantage to a longer name? You could just update the description for the model. 
As [mentioned in the RFC](https://vendasta.jira.com/wiki/spaces/RD/pages/825327688/Exposing+Invoices+to+Partners+via+API?focusedCommentId=851181883#comment-851181883), I'm not sure if having the customer id on each line item makes sense for a generic payable invoice but I'll leave the decision up to the @vendasta/psycho-billers.

Comment (coreyhickson) [reply openapi/platform/platform.yaml:711]: I think the benefit of AccountsPayable vs Payable is that AccountsPayable is a more common term in accounting (for example, in [QuickBooks](https://developer.intuit.com/app/developer/qbo/docs/api/accounting/all-entities/apagingdetail), although they say AP & AR). That being said, it's a different domain than what is with ChargeBee, so I'm not an expert here.

**Review by richard-rance — APPROVED** [review]

**Pull request merged**